hhs_oauth_server/urls.py

```diff
@@ -36,6 +36,7 @@
         url(r'^mymedicare/', include('apps.mymedicare_cb.urls')),
     ]
 
-urlpatterns += [
-    url(r'^$', home, name='home'),
-]
+if not getattr(settings, 'NO_UI', False):
```

Comment:
What is this meant to turn off? `NO_UI` implies that it would be for disabling all of the developer / app management pages, is that right?

Reply:
Yes, in the long term, that's exactly what it's meant to do long term.